net/acme/files/run.sh

```diff
@@ -215,6 +215,7 @@ issue_cert()
 
     if echo $keylength | grep -q "^ec-"; then
         domain_dir="$STATE_DIR/${main_domain}_ecc"
+        acme_args="$acme_args --ecc"
```

Comment:
Hmm, as I'm reading the acme.sh documentation, this should only be specified on renew, right? So I don't think we can just set it unconditionally?